app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h3>All Contestants</h3>
+  <% @contestants.each do |contestant| %>
+    <div id="contestant-<%= contestant.id %>">
+      <p>Name: <%= contestant.name %></p>
+      <% contestant.projects.each do |project| %>
+        <p>Projects: <%= project.name %></p>
```

Comment:
This is more of a nit, but the user story displays a comma separated list, so to be fully loyal to that user story's spec, you could instead do `contestant.projects.join(',')` to create that list rather than iterating